src/Features/Core/Portable/FindSymbols/SymbolTree/SymbolTreeInfoIncrementalAnalyzer.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Remote;
+using Microsoft.CodeAnalysis.SolutionCrawler;
+
+namespace Microsoft.CodeAnalysis.FindSymbols.SymbolTree
+{
+    internal partial class SymbolTreeInfoIncrementalAnalyzerProvider
+    {
+        private class SymbolTreeInfoIncrementalAnalyzer : IncrementalAnalyzerBase
```

Comment:
this incremental-analyer now runs in proc, but shells out to OOP to perform the 3 individual steps it supports.  This is morally the same, but it does move us closer to not having solution-crawler in OOP at all.